core-services/prow/02_config/_config.yaml

```
@@ -117,8 +117,18 @@ deck:
         slug: redhat-chaos
   - repo: openshift/operator-framework-operator-controller
     rerun_auth_configs:
-      github_users:
-      - m1kola
+      github_team_slugs:
+      - org: openshift
+        slug: openshift-team-operator-ecosystem
+      - org: openshift
+        slug: openshift-team-operator-runtime
+  - repo: openshift/operator-framework-olm
+    rerun_auth_configs:
+      github_team_slugs:
+      - org: openshift
+        slug: openshift-team-operator-ecosystem
+      - org: openshift
+        slug: openshift-team-operator-runtime
```

I presume these team names come from dynamic config?  Asking, since we haven't had those teams in some time, and I want to poke the source for an update if we know it.

---

Yes, those ought to be github groups

---

So, does this mean it should just be `openshift-team-operator-runtime`?

---

related to https://github.com/openshift/org/pull/251

---

Once above merges, then this should change to just
```suggestion
        slug: openshift-team-operator-framework
```